README.md

```diff
@@ -48,6 +48,22 @@ git clone https://github.com/catppuccin/grub.git && cd grub
 sudo cp -r src/* /usr/share/grub/themes/
 ```
 
+⚠️ If you use encrypt your `/` partition (for exemple with luks) Grub wont be able to reach files in `/usr/share/grub/themes/` ⚠️
+
+You probably have a separate `/boot` partition, as an alternative you can copy the files to `/boot/grub/themes`
+
+```shell
+sudo mkdir /boot/grub/themes
+sudo cp -r src/* /boot/grub/themes/
+```
+
+For Fedora:
+```shell
+sudo mkdir /boot/grub2/themes
+sudo cp -r src/* /boot/grub2/themes/
+```
+
```

Comment:
You do need to update the `/etc/default/grub` file accordingly however, as this will still not allow it to go into the /usr/share/grub/themes directory.
You would have to change the line to `GRUB_THEME="/boot/grub2/theme/CATPPUCCIN-THEME/theme.txt"`

Reply:
Sorry for taking so long to get to this PR! Basically in the README, this is already covered with ``Q: "Grub isn't detecting the theme"`` which tells the user to replace `/usr/share/` with `/boot/` so that applies to encrypted use cases. Therefore, I will not be merging.